client/src/app/components/page/list/list.component.scss

```diff
@@ -7,4 +7,18 @@
     justify-content: center;
     margin-top: 10px;
     align-items: center;
-}
\ No newline at end of file
+}
+
+li a{
```

Comment:
Remove all css from here. You can use bootrap default elements and styling.

Reply:
Removed